fury/interactor.py

```diff
@@ -331,10 +401,17 @@ def _callback(_obj, event_name):
             # Update event information.
             interactor_ = self.GetInteractor()
             if interactor_ is not None:
-                self.event.update(event_name, interactor_)
+                # self.event.update(event_name, interactor_)
                 callback(self, prop, *args)
             else:
                 print('interactor is none')
                 print('event name is', event_name)
 
+        if vtk.vtkCommand.GetEventIdFromString(event_type) == 0:
+            if event_type not in self.event2id:
+                self.event2id[event_type] = \
+                    vtk.vtkCommand.UserEvent + len(self.event2id) + 1
+
+            event_type = self.event2id[event_type]
```

Comment:
Needs a comment describing this section: dealing with custom events not defined in VTK.